### PR TITLE
New app: Zenhub

### PR DESCRIPTION
**Pull request by thiagobrez**
Integrate your [Zenhub](https://www.zenhub.com/) workspace to the Tidbyt display.

Zenhub is a productivity management tool to keep track of issues and features like Jira, Asana, Azure DevOps. Using your Zenhub API Keys, we can pull information from your workspace and show in your Tidbyt display.

- Choose a pipeline to display issues (tickets)
- Filter by labels
- Filter by assignees

zenhub

- In the top row
  - Show the chosen pipeline. Can be any pipeline in your Zenhub workspace. In this example: "Current Sprint"
- In center
  - Show the first 3 issues (tickets) in the pipeline. Issue number and title

### CODE REVIEW

**Comment by thiagobrez** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by thiagobrez** [comment]
Included MD5 hash of user's api key in the cache key, per comments in other PRs

**Review by matslina — CHANGES_REQUESTED** [review]
Hey, thanks for the PR!

I've added a few comments in the code, and of course the build is failing but that should be a simple fix I think. When we get it merged, feel free to ping me on discord and I'll get you set up with beta access so you can test it out in production before we publish it. I miiiight set up my own Zenhub account for testing, but maybe not. =)

File: apps/zenhub/zenhub.star [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+def main(config):
+    zenhub_rest_api_key = config.str("zenhub_rest_api_key", None)
+    zenhub_gql_api_key = config.str("zenhub_gql_api_key", None)
+    workspace_id = config.str("workspace_id", None)
+    repo_id = config.get("repo_id", None)
+    selected_pipeline = config.str("selected_pipeline", None)
+    selected_labels = config.str("selected_labels", "")
+    selected_assignees = config.str("selected_assignees", "")
+
+    if not zenhub_rest_api_key:
```
[the remark this answers is not in the document]
Comment (thiagobrez) [reply apps/zenhub/zenhub.star:178]: Hey, Mats! Thanks for the review. Nice suggestion, I added a welcome page with fake tasks guiding the user to set-up the configuration info, like below:

https://user-images.githubusercontent.com/26878038/193160949-91a31a78-12fd-4c57-bf56-42236f915f53.mov

File: apps/zenhub/zenhub.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+var source []byte
+
+// New creates a new instance of the Zenhub applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "zenhub",
+		Name:        "Zenhub",
+		Author:      "thiagobrez",
+		Summary:     "Show Zenhub issues",
+		Desc:        "Show Zenhub issues from a Pipeline of your choice. Filter by labels and assignees.",
```
[the remark this answers is not in the document]
Comment (thiagobrez) [reply apps/zenhub/zenhub.go]: Yeah, I imagined this could be a thing. I moved the guides to the description, see if that works!

**Comment by thiagobrez** [comment]
Fixed the build and the other requested changes. Let me know what you think!

**Review by matslina — APPROVED** [review]

**Pull request merged**